addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -3298,5 +3298,12 @@ export function isMacOS() {
 export function cleanZWS(node) {
     [node, ...descendants(node)]
         .filter(node => node.nodeType === Node.TEXT_NODE && node.nodeValue.includes('\u200B'))
-        .forEach(node => node.nodeValue = node.nodeValue.replace(/\u200B/g, ''));
+        .forEach((node) => {
+            if (node.nodeValue === "\u200B") {
+                // Remove text nodes that contain only zws
+                node.parentElement.removeChild(node);
+            } else {
+                node.nodeValue = node.nodeValue.replace(/\u200B/g, "");
+            }
+        });
```

Comment:
```suggestion
        .forEach((node) => {
           node.nodeValue = node.nodeValue.replace(/\u200B/g, "");
           isEmptyBlock(node) && node.remove();
        });
```

Can we do something like this instead?